test/lib/cas_attributes_test.rb

```diff
@@ -2,6 +2,8 @@
 
 class CasAttributesTest < ActiveSupport::TestCase
   setup do
+    ENV['CAS_ACCESS_TOKEN'] = 'test'
```

Comment:
If you like you can create a `.env.test` file to accommodate ENV vars needed in testing.